tasks/testing.rake

```diff
@@ -214,6 +214,15 @@ module MinitestSuite
   end
 end
 
+def execute_cmd_with_env(env, cmd)
+  if RUBY_VERSION.to_f > 2
```

Comment:
I didn't find exactly when `Kernel.system` was introduced but the Ruby 2.0.0 documentation shows this method: `system([env,] command... [,options]) → true, false or nil`
https://ruby-doc.org/core-2.0.0/Kernel.html#method-i-system

> shouldn't this be >= 2 to include 2.0 (I'm assuming env support was introduced since 2.0)
If that's true you can safely assume we're targeting 2.0 only, and leaving older releases for < 2.0 (I'll update the gemspec soon to reflect that).

You mean removing the condition and use `Kernel.system` all the time since we're targeting 2+ only ? Or just replace `> 2` with `>= 2` ?